src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,37 @@
+.stop-watch {
+  position: relative;
+  border: 2px dotted grey;
+  border-radius: 50%;
```

Comment:
It's better to use a variable for the border-color to make it easier to modify later.